apps/api/src/routes/v1_identities_updateIdentity.happy.test.ts

```diff
@@ -124,6 +124,7 @@ test("sets new ratelimits", async (t) => {
     where: (table, { eq }) => eq(table.identityId, identity.id),
   });
 
+  console.log({ found });
```

Comment:
You left this here

Reply:
no that's on purpose, I just added it, cause this test only fails on github, not local